homeassistant/components/elevenlabs/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The ElevenLabs text-to-speech integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from elevenlabs.client import AsyncElevenLabs
+from elevenlabs.core import ApiError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+
+from .const import CONF_MODEL, DEFAULT_MODEL
+from .tts import get_model_by_id
+
+PLATFORMS: list[Platform] = [Platform.TTS]
+
+
+@dataclass(kw_only=True, slots=True)
+class ElevenLabsData:
+    """ElevenLabs data type."""
+
+    client: AsyncElevenLabs
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up ElevenLabs text-to-speech from a config entry."""
+    entry.add_update_listener(update_listener)
+    client = AsyncElevenLabs(api_key=entry.data[CONF_API_KEY])
+    model_id = entry.options.get(CONF_MODEL, entry.data.get(CONF_MODEL))
```

Comment:
Why is present in either options or data? Let's just pick one place for it to be and just use `entry.something.get(CONF_MODEL, DEFAULT_MODEL)`

Reply:
that should be options, as users can change it.